src/engines/L3/element.js

```diff
@@ -465,6 +465,11 @@ const propsTransformer = {
     }
   },
   set 'inspector-data'(v) {
+    // Skip processing if inspector is not enabled for performance optimization
+    if (!Settings.get('inspector', false)) {
```

Comment:
micro optimization, but let's change this from a `falsey` check to a `false` check (`Settings.get('inspector', false) === false`)